google-cloud-core/src/main/java/com/google/cloud/GrpcTransportOptions.java

```diff
@@ -281,22 +261,34 @@ public int getMaxTimeout() {
     return maxTimeout;
   }
 
+  public Builder toBuilder() {
+    return new Builder(this);
+  }
+
   @Override
-  protected int baseHashCode() {
-    return Objects.hash(super.baseHashCode(), executorFactoryClassName, initialTimeout,
+  public int hashCode() {
```

Comment:
It has to do with the inheritance setup in `google-cloud-java`. The pattern is to use a `baseHashCode` function in abstract classes and `hashCode` in concrete classes (which call the `baseHashCode` of the parent class). Since this class is converting from abstract to concrete, that's why you are seeing this change.